haiku/_src/moving_averages_test.py

```diff
@@ -53,6 +53,17 @@ def test_warmup(self):
     np.testing.assert_allclose(
         (random_input[0] + random_input[1]) / 2, ema(random_input[1]))
 
+  def test_invalid_warmup_length(self):
```

Comment:
I think this needs `@test_utils.transform_and_run` too.

Reply:
ok :)